Modules/AuctionManager/GUI.lua

```diff
@@ -289,6 +292,11 @@ function AuctionManagerGUI:GenerateAuctionOptions()
                 self.awardPlayer = ""
                 self:Refresh()
             end),
+            confirm = (function()
+                return "Are you sure, you want to award " .. self.itemLink .. 
```

Comment:
Lets use `string.format`. There should be one as an example in the `AuctionManager:Award`